utils/catchAsync.util.js

```diff
@@ -0,0 +1,7 @@
+const catchAsync = fn => {
```

Comment:
Sigo sin entender para que usas esto?

Reply:
Lo puse porque intenté levantar el servidor y no me dejaba porque faltaba este archivo, así que fui a uno de los PR que no se habían resuelto y lo agregué dentro de mi código. Sin embargo esto no hace parte de mi tarea y está dentro del commit de refactor to run app